apps/browser/components/modals/SignInModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import Modal from "./ModalBase";
+import Button from "../Button";
+import { SignOut } from "@phosphor-icons/react";
+import Image from "next/image";
+import { signIn, signOut, useSession } from "next-auth/react";
+import { useAccountConfig } from "@/lib/hooks/useAccountConfig";
+import { useAtom } from "jotai";
+import { localOpenAiApiKeyAtom } from "@/lib/store";
+
+interface AccountConfigProps {
+  apiKey: string | null;
+  allowTelemetry: boolean;
```

Comment:
Shouldn't we get these from the atoms?

Reply:
https://github.com/polywrap/evo.ninja/pull/555#discussion_r1427998324